cmd/gaia/cmd/gaiareplay/main.go

```diff
@@ -127,7 +127,9 @@ func run(rootDir string) {
 	if err != nil {
 		panic(err)
 	}
-	defer proxyApp.Stop()
+	defer func() {
+		_ = proxyApp.Stop()
+	}()
```

Comment:
confused as to why this is necessary?

Reply:
should we not be checking this error still though? 
```
	defer func() {
		err = proxyApp.Stop()
		if err != nil {
			panic(err)
		}
	}()
```

Reply:
I guess not actually because the error is only returned if the proxyApp is already stopped, which wouldn't be an issue in this case